### DISCUSSION TITLE
19차 모임 (9/14 or 9/19) 발표 주제 제안

### DISCUSSION THREAD

**Discussion by haksungjang** [OpenChain-KWG/meeting#3] [General]
OpenChain Korea Work Group 19차 모임이 현대오토에버에서 9/14(목) or 9/19(화)에 예정되어 있습니다. (일정은 8월초 fix 예정)

- 이 때 주제 발표를 원하시는 분은 발표 제목과 간단한 내용을 comment로 남겨주시기 바랍니다.
- 혹시, (내가 발표하지는 못하지만) 누군가가 공유해주기를 원하시는 주제가 있어도 comment 남겨주세요.

감사합니다.

**Comment by haksungjang** [comment]
지난 모임에서 [제안](https://github.com/OpenChain-KWG/meeting/discussions/2#discussioncomment-5890721)되었던 LG전자의 "글로벌 제조업체 세계 최초 오픈소스 보안 관리 체계 준수" 사례 발표를 다시 요청 드립니다. @k2heart

**Reply by jungsp** [reply]
저도 찬성합니다.^^

**Reply by k2heart** [reply]
LG전자 정재욱 책임님이 발표해주실 예정입니다. 
발표 제목 : LG전자 제품 보안 관리 체계 기반 오픈소스 보안 보증 표준 준수 사례 소개
발표 시간 : 20-30분

**Reply by haksungjang** [reply]
@k2heart 님, 
발표 시간이 15:00~15:20 로 확정되었음을 발표자 분께 안내 부탁 드립니다. :) 
감사합니다!

**Comment by haksungjang** [comment]
라인의 [ISO/IEC 5230 인증 선언](https://linepluscorp.com/pr/news/ko/2023/4612) 사례 발표를 요청 드립니다. @syleeeee 

발표 제목과 예상 소요 시간을 남겨주시면 고맙겠습니다. :)

**Reply by syleeeee** [reply]
네, 발표제목은 팀에서 상의해보고 최종 결정하겠습니다. 
가제: 우당탕탕 LINE의 OpenChain 인증 여정
예상 소요시간: 30분

**Reply by haksungjang** [reply]
'우당탕탕' ㅋㅋㅋㅋㅋ
기대됩니다!!!

**Reply by jungsp** [reply]
오~ "우당탕탕" 정말 재밌는 제목이 되겠네요. 제 발표 제목도 다시 재밌는 것으로 고민해 볼께요.^^

**Reply by yoobato** [reply]
!! 안그래도 ㅠㅠ 저희는(결제원은) 4분기 발표로 부탁 말씀드리랴고 했는데...!!😭

이번에 멋지게 발표해주시면 참고해서 저희도 멋진 ISO/IEC 5230:2020 인증 선언 후기로 꼭 발표드리겠습니다 🙏

**Comment by syleeeee** [comment]
요즘 AI와 관련된 신규 라이선스들이 등장하고 있습니다. 
오픈소스 라이선스라고 볼 수 없다는 의견도 많은데요. 이 계열 라이선스들의 특징과 의무사항 요약 공유하는 세션이 있어도 흥미로울 것 같습니다. 
https://blog.opensource.org/metas-llama-2-license-is-not-open-source/
https://huggingface.co/blog/open_rail

**Reply by jungsp** [reply]
흥미롭겠네요.^^ 저도 살펴보겠습니다.

**Reply by soimkim** [reply]
인공지능 라이선스 관련하여 LG전자 최혜성 책임님께서 발표하실 예정입니다. 😃

**Reply by soimkim** [reply]
9/14는 LG전자 교육이 있어 참여 불가입니다.

**Reply by syleeeee** [reply]
@soimkim 님, 감사합니다. 발표 시간도 정해지면 공유 부탁드리겠습니다~!

**Reply by soimkim** [reply]
30분입니다.

**Comment by jungsp** [comment]
저 "Legal Subgroup 소개"로 발표 요청드립니다. 시간은 길게 안 주셔도 될 것 같아요.^^ 감사합니다.

**Comment by syleeeee** [comment]
이번 모임 발표 지원은 시간상 여기까지 받는 것이 좋을 것 같습니다!

**Reply by haksungjang** [reply]
슬랙에서는 공지를 했지만, (날짜가 확정되지 않은 관계로) 아직 메일링리스트로 발표 자원 안내를 하지 못했습니다. 공지를 하고 나면 혹시 추가로 자원/요청하는 건이 있을 수 있습니다. 모임 시간 상 발표 세션이 많다면, 운영위원회에서 논의하여 조절할 수 있을 것 같습니다. :)

**Comment by hyundaiautoever** [comment]
안녕하세요. 19차 모임 주제 관련해서 장학성님과 논의를 했고, 아래의 내용에 대해 발표진행을 요청드리오니, 검토 부탁드립니다.

1) 현대차그룹 오픈소스 리스트 공개를 통한 중소/중견기업 지원방안(현대오토에버 류창한책임) / 20분
2) 현대차그룹 오픈소스관리시스템 구축사례(현대자동차 이창우책임, 현대모비스 이영준 연구, 현대오토에버 이지현 책임) /  30분(회사별10분)
3) 패널토론(현대차, SK, LG전자, 카카오, 삼성전자, LINE, ETRI etc. 전문가 분들이 참여하여 특정주제에 대해 패널토론이 진행되면 좋겠습니다.)

**Reply by haksungjang** [reply]
넵, 의견 감사합니다. 차주 월요일(8/21) 운영위원회 미팅 시 적극 반영하겠습니다. ^^
